provisioning/roles/classyllama.nginx/molecule/default/converge.yml

```diff
@@ -3,7 +3,7 @@
   hosts: all
   become: true
   vars:
-    nginx_diffie_hellman_bits: 1024
```

Comment:
This is in the classyllama.nginx ansible role and should not be modified here, but instead in the repo (https://github.com/classyllama/ansible-role-nginx) for that ansible role and then the role updated here in the DevEnv using ansible-galaxy as specified in the requirements.yml file.

Reply:
Other pull request for https://github.com/classyllama/ansible-role-nginx :

https://github.com/classyllama/ansible-role-nginx/pull/3 